pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -571,3 +571,6 @@ proxy:
     # Sets the envoy runtime value to limit maximum number of incoming
     # connections to a builtin gateway data plane proxy
     globalDownstreamMaxConnections: 50000 # ENV: KUMA_PROXY_GATEWAY_GLOBAL_DOWNSTREAM_MAX_CONNECTIONS
+
+policies:
```

Comment:
What's the motivation for this being a user configurable option?

Reply:
User can define their own plugins and order in case new ones, also it's possible to disable one plugin when they find some bug that impacts configuration.

Reply:
Can they define their own plugins though right now without rebuilding Kuma? If so, can't they just enable it in the source?

I feel like policies are core components. There can always be bugs in any part of Kuma but I'm not sure that means users should be able to disable those parts individually.

Did we have a concrete use case or user request for this?

Reply:
Also adding a user configurable ordering doesn't really make sense anyway. The whole point of ordering them a certain way is that they depend on each other, isn't it?

Reply:
I guess @michaelbeaumont has a point here. Maybe we're just trying to overthink things. There might be a place in the future for this to be configurable but it's not now.

Reply:
Let's just have the array in `pkg/plugins/policies`